app/src/main/kotlin/com/example/android/picover/MainActivity.kt

```diff
@@ -32,18 +38,25 @@ class MainActivity : ComponentActivity() {
 					NavigationItem.Camera,
 					NavigationItem.Profile
```

Comment:
Remove the import of `Profile` or import all navigation items for consistency.

Reply:
of course, fixed :) thanks !

Reply:
@michal-kucznerowicz  please check my changes :)